docs/arcadia-users-group/20231031-intro-to-r/lesson.md

```diff
@@ -0,0 +1,1674 @@
+```
+
+```
+## # A tibble: 1,438 × 13
+##    record_id month   day  year plot_id species_id sex   hindfoot_length weight
+##        <dbl> <dbl> <dbl> <dbl>   <dbl> <chr>      <chr>           <dbl>  <dbl>
+##  1      9790     1    19  1985      16 RM         F                  16      4
+##  2      9791     1    19  1985      17 OT         F                  20     16
+##  3      9792     1    19  1985       6 DO         M                  35     48
+##  4      9793     1    19  1985      12 DO         F                  35     40
+##  5      9794     1    19  1985      24 RM         M                  16      4
+##  6      9795     1    19  1985      12 DO         M                  34     48
+##  7      9796     1    19  1985       6 DM         F                  37     35
+##  8      9797     1    19  1985      14 DM         M                  36     45
+##  9      9798     1    19  1985       6 DM         F                  36     38
+## 10      9799     1    19  1985      19 RM         M                  16      4
+## # ℹ 1,428 more rows
+## # ℹ 4 more variables: genus <chr>, species <chr>, taxa <chr>, plot_type <chr>
+```
+
+The `==` sign means "is equal to."
+There are several other operators we can use: >, >=, <, <=, and != (not equal to).
+Another useful operator is `%in%`, which asks if the value on the left hand side is found anywhere in the vector on the right hand side.
+For example, to get rows with specific `species_id` values, we could run:
+
+
+```r
+filter(surveys, species_id %in% c("RM", "DO"))
+```
+
+```
+## # A tibble: 2,835 × 13
+##    record_id month   day  year plot_id species_id sex   hindfoot_length weight
+##        <dbl> <dbl> <dbl> <dbl>   <dbl> <chr>      <chr>           <dbl>  <dbl>
+##  1        68     8    19  1977       8 DO         F                  32     52
+##  2       292    10    17  1977       3 DO         F                  36     33
+##  3       294    10    17  1977       3 DO         F                  37     50
+##  4       311    10    17  1977      19 RM         M                  18     13
+##  5       317    10    17  1977      17 DO         F                  32     48
+##  6       323    10    17  1977      17 DO         F                  33     31
+##  7       337    10    18  1977       8 DO         F                  35     41
+##  8       356    11    12  1977       1 DO         F                  32     44
+##  9       378    11    12  1977       1 DO         M                  33     48
+## 10       397    11    13  1977      17 RM         F                  16      7
+## # ℹ 2,825 more rows
+## # ℹ 4 more variables: genus <chr>, species <chr>, taxa <chr>, plot_type <chr>
+```
+
+We can also use multiple conditions in one `filter()` statement.
+Here we will get rows with a year less than or equal to 1988 and whose hindfoot length values are not `NA`.
+The `!` before the `is.na()` function means "not".
+
+
+```r
+filter(surveys, year <= 1988 & !is.na(hindfoot_length))
```

Comment:
Not sure if it comes up later, could be good to show `drop_na()` for getting rid of any rows where any column has an NA

Reply:
👍 will add it to the extras section at the end!